gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetTest.java

```diff
@@ -253,15 +263,19 @@ public void testGenerateCopyEntitiesMultiSnapshotWhenDestEmpty() throws IOExcept
     FileSystem sourceFs = sourceBuilder.build();
 
     IcebergTable icebergTable = MockIcebergTable.withSnapshots(Arrays.asList(SNAPSHOT_PATHS_1, SNAPSHOT_PATHS_0));
+    IcebergTable targetTable = MockIcebergTable.withSnapshots(Arrays.asList(SNAPSHOT_PATHS_2));
     IcebergDataset icebergDataset =
-        new TrickIcebergDataset(testDbName, testTblName, icebergTable, new Properties(), sourceFs);
+        new TrickIcebergDataset(testDbName, testTblName, icebergTable, targetTable, new Properties(), sourceFs);
 
     MockFileSystemBuilder destBuilder = new MockFileSystemBuilder(DEST_FS_URI);
     FileSystem destFs = destBuilder.build();
 
     CopyConfiguration copyConfiguration =
         CopyConfiguration.builder(destFs, copyConfigProperties).preserve(PreserveAttributes.fromMnemonicString(""))
             .copyContext(new CopyContext()).build();
+    try (MockedConstruction<PostPublishStep> mockedPostPublishStep = mockConstruction(PostPublishStep.class)) {
+      PostPublishStep step = new PostPublishStep(icebergDataset.getFileSetId(), Maps.newHashMap(), new IcebergRegisterStep(icebergTable, targetTable), 0);
```

Comment:
I thought the constructor needed to be mocked since I was running into issues while testing for PostPublishStep not being able to instantiate. Debugging further helped me understand that this step can be avoided and all we need is to use `mockito-inline` instead of `mockito-core` and then it resolved the issue for me